### ISSUE TITLE
SIGABRT When Running From XCode

### ISSUE THREAD

**Issue by owenlejeune** [DevUtilsApp/DevUtils-app#75]
I've setup signing with my personal ID but running the app from XCode is giving a SIGABRT 0__abort_with_payload on Thread 1. I'm also getting this output:

> dyld: Library not loaded: @rpath/ShortcutRecorder.framework/Versions/A/ShortcutRecorder
  Referenced from: /Users/owenlejeune/Library/Developer/Xcode/DerivedData/DevUtils-byofmcbxhvpfezasyyfoughnsluu/Build/Products/Debug/DevUtils.app/Contents/MacOS/DevUtils
  Reason: no suitable image found.  Did find:
	/Users/owenlejeune/Library/Developer/Xcode/DerivedData/DevUtils-byofmcbxhvpfezasyyfoughnsluu/Build/Products/Debug/DevUtils.app/Contents/MacOS/../Frameworks/ShortcutRecorder.framework/Versions/A/ShortcutRecorder: code signature in (/Users/owenlejeune/Library/Developer/Xcode/DerivedData/DevUtils-byofmcbxhvpfezasyyfoughnsluu/Build/Products/Debug/DevUtils.app/Contents/MacOS/../Frameworks/ShortcutRecorder.framework/Versions/A/ShortcutRecorder) not valid for use in process using Library Validation: mapped file has no Team ID and is not a platform binary (signed with custom identity or adhoc?)

**Comment by trungdq88** [comment]
Hi there. Have you try to run `carthage bootstrap --platform macOS`?

**Comment by owenlejeune** [comment]
Yes, I ran `carthage bootstrap --platform macOS` and am still seeing this issue

**Comment by owenlejeune** [comment]
@haidarzhy that fixed my issue. Thanks!
